submissions/odunayoFeyiAkinbile/HTML/contact-form.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Contact Form</title>
+    </head>
+    <body>
+        <h1>Contact Form</h1>
+        <form action="#" method="POST" enctype="multipart/form-data">
+            <nav>
+                <a href="../html/product-page.html">Product</a>
+                <a href="../html/contact-form.html">Contact Form</a>
```

Comment:
This link will not work. The folder name is incorrect

Reply:
yes when i clone dufuna-fem repo the folder is not there so I create it again